### PR TITLE
upgrade to Falco 0.30.0

### PR DESCRIPTION
**Pull request by leogr**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

/kind documentation

> /kind failing-test

/kind feature

> If this PR will release a new chart version please make sure to also uncomment the following line:

/kind chart-release

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

/area falco-chart

> /area falco-exporter-chart

> /area falcosidekick-chart

**What this PR does / why we need it**:

Still a wip

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Checklist**

- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] CHANGELOG.md updated

### CODE REVIEW

**Review by Issif — COMMENTED** [review]
I didn't saw `kubernetesSupport.enableNodeFilter` in default from `values.yaml`, can you add it please.

File: falco/README.md [hunk 1]
```diff
@@ -65,6 +65,7 @@ The following table lists the configurable parameters of the Falco chart and the
 | `kubernetesSupport.enabled`        | Enable Kubernetes meta data collection via a connection to the Kubernetes API server                                                                                                                       | `true`
 | `kubernetesSupport.apiAuth`        | Provide the authentication method Falco should use to connect to the Kubernetes API                                                                                                                        | `/var/run/secrets/kubernetes.io/serviceaccount/token`
 | `kubernetesSupport.apiUrl`         | Provide the URL Falco should use to connect to the Kubernetes API                                                                                                                                          | `https://$(KUBERNETES_SERVICE_HOST)`
+| `kubernetesSupport.enableNodeFilter`  | If true, only the current node (on which Falco is running) will be considered when requesting metadata of pods                                                                                          | `https://$(KUBERNETES_SERVICE_HOST)`
```
Comment (leogr) [inline falco/README.md:68]: The default value should be `true`. I'm fixing

**Comment by leogr** [comment]
> I didn't saw `kubernetesSupport.enableNodeFilter` in default from `values.yaml`, can you add it please.

Hey @Issif 

It's here :point_down: 
https://github.com/falcosecurity/charts/pull/277/files#diff-27a4a6615a33a2df2f95950e079c0c90701f4bcb87d1d080c59dd23f85b5da58R37

Anyways, I have found a small error in the README.md (see the above comment), I'm fixing it.

**Comment by leogr** [comment]
Done, PTAL :)

**Review by Issif — APPROVED** [review]

**Pull request merged**